### PR TITLE
Milestone1

### PR DESCRIPTION
**Pull request by Arinpe**
In this milestone is the first step to creating the calculator app as seen in the Microver-Odin project

### CODE REVIEW

File: milestone1/src/App.css [hunk 1]
```diff
@@ -0,0 +1,39 @@
+.App {
```
Comment (bellom) [inline milestone1/src/App.css]: We don't need to create a new directory, `src/App` is ok.

And also let's delete unnecessary files and logos.

image

**Review by bellom — CHANGES_REQUESTED** [review]
## Hi @Arinpe  :clap: 

### Awesome job so far!! :+1: 

#### You have done wonderfully well, you passed `almost` all the checklist for this project :+1: just a few issues left to work on............ :slightly_smiling_face: 

# Status: Changes Required! ♻️ 

**1. Kindly check my comments under this review;**

**2. Your app seems not working on a local machine, I think we are having a problem with the react directory, you don't need to create a new folder called `milestone1`, kindly revert and try to run your app again.**

image


**3. README file doesn't appear on the file changes, so I'm forced to drop the comment here: kindly update your README file with the correct information, seems like you have Weather app setup, kindly use the normal setup for react app i.e `npm i` then `npm start`, and also kindly add the correct live demo link, the added seems broken.**

image


**4. Let's improve your PR description, check out the example below**
image
 

### ♻️ Kindly re-submit for review after making all those changes :slightly_smiling_face: Thumps -up await you.....

**Best**

**Comment by bellom** [comment]
@Arinpe I can see your effort trying to fix the linter.

Please update your README with the correct information as I said above so we can get this PR approved, you won't need to send in another CRR.
And also update your live demo link with the correct link, the current one seems not working.

I will be on this PR for few mins.

**Comment by Arinpe** [comment]
> @Arinpe I can see your effort trying to fix the linter.
> 
> Please update your README with the correct information as I said above so we can get this PR approved, you won't need to send in another CRR.
> And also update your live demo link with the correct link, the current one seems not working.
> 
> I will be on this PR for few mins.

I don't quite understand what you want me to do with the read me file because everything seems fine to me

**Comment by bellom** [comment]
> > @Arinpe I can see your effort trying to fix the linter.
> > Please update your README with the correct information as I said above so we can get this PR approved, you won't need to send in another CRR.
> > And also update your live demo link with the correct link, the current one seems not working.
> > I will be on this PR for few mins.
> 
> I don't quite understand what you want me to do with the read me file because everything seems fine to me

image


👆 perhaps a screenshot would help.

and also kindly confirm your live demo link and share it with me here.

**Comment by Arinpe** [comment]
I am done with the requested changes

**Review by bellom — APPROVED** [review]
## Hi @Arinpe  :clap: 

To Improve:

README file:
React set up example:
```
- git clone `your repo`
- cd into the folder
- git checkout `branch-name`
- run `npm i`
- run `npm start`
```

# Status: Approved

**Best**

**Comment by bellom** [comment]
## Hi @Arinpe  👏  

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

CR can't review a merge PR, kindly create a new one.

**Pull request merged**